src/core/ddsi/include/dds/ddsi/q_xmsg.h

```diff
@@ -61,15 +61,11 @@ struct iceoryx_header {
    uint32_t data_size;
    unsigned char data_kind;
    ddsi_keyhash_t keyhash;
+   bool does_contain_serialized_data;
```

Comment:
`contains_serialized_data`? or something shorter, but more importantly:

It appears not to be set anywhere as of now, is this correct?

I will then set it when I need to determine whether the data is serialized and has to be deserialized (for https://github.com/eclipse-cyclonedds/cyclonedds/issues/941).

Could currently be set to false whenever we fill the iceoryx chunk (but we will evaluate it nowhere until https://github.com/eclipse-cyclonedds/cyclonedds/issues/941).

Reply:
Or just `serialized` would work too. Perhaps make it an enum eventually because of the various serialization options?

Reply:
@k0ekk0ek @MatthiasKillat

I like expressive variable names and prefer a `is`, `has`, `does` ... prefix for boolean variable names. If one reads such variables later in the code during a review the reviewer knows instantly that a boolean is handled here. Therefore, I suggest to leave the name as it is.

At the moment we are not planning to have multiple serialization options and a bool seems sufficient at the moment. If this changes in the future we can still adapt it.